client-registry-santempi/package-metadata.json

```diff
@@ -6,9 +6,8 @@
   "version": "0.0.1",
   "dependencies": [],
   "environmentVariables": {
-    "STATEFUL_NODES": "single",
-    "SANTE_POSTGRES_PASSWORD": "postgres",
     "SANTEMPI_INSTANCES": "1",
+    "SANTE_POSTGRES_PASSWORD": "postgres",
```

Comment:
Did we add this to the docker compose file ?

Reply:
yep 
![image](https://user-images.githubusercontent.com/48720958/212656445-b379d50f-2b95-44c4-84c3-d96a2c778aa0.png)